solr/core/build.gradle

```diff
@@ -192,4 +192,8 @@ dependencies {
   })
 }
 
-
+test {
+  testLogging {
+    showStandardStreams = true
```

Comment:
I would expect to see this change in an existing test section, which makes me think that is probably the reason it isn't working.

Reply:
So, we'll want to look to see where the existing test configuration lies. I see some configuration options in the main `build.gradle` file: https://github.com/ponder-lab/solr/blob/552863112e89845fdb34727e7b3e06658d171c70/build.gradle#L123-L132

Reply:
Let's have a look at the default one: https://github.com/ponder-lab/solr/blob/552863112e89845fdb34727e7b3e06658d171c70/build.gradle#L127

Reply:
It seems they put all of their Gradle configuration stuff in the `gradle` directory. And, it looks like there's a `testing` folder there. Looking at the following, it seems there's a verbose option for testing, which sounds perfect:

https://github.com/ponder-lab/solr/blob/552863112e89845fdb34727e7b3e06658d171c70/gradle/testing/defaults-tests.gradle#L35

We can pass that option to Gradle using `-D` to specify the property, e.g., `-Dtests.verbose=true`.

Reply:
And we can see that streams are off by default here: https://github.com/ponder-lab/solr/blob/287f51efaab0e27afe050521789b2a43fc8a6356/gradle/testing/defaults-tests.gradle#L167